sdk/vaa/payloads.go

```diff
@@ -257,6 +264,24 @@ func (r BodyTokenBridgeUpgradeContract) Serialize() []byte {
 	return serializeBridgeGovernanceVaa(r.Module, ActionUpgradeTokenBridge, r.TargetChainID, r.NewContract[:])
 }
 
+func (r BodyRecoverChainId) Serialize() []byte {
+	// Module
+	buf := LeftPadBytes(r.Module, 32)
+	// Action
+	var action GovernanceAction
+	if r.Module == "Core" {
+		action = ActionCoreRecoverChainId
+	} else {
+		action = ActionTokenBridgeRecoverChainId
```

Comment:
What about NFT bridge? If they both use the same enum, should it be named something that indicates that? Or, if NFT bridge is not supported, check for anything other than "TokenBridge" and throw an error?

Reply:
`"NFTBridge"` is supported and it has the same action as the token bridge `3`. Fwiw the other payloads in this file (e.g. `BodyTokenBridgeUpgradeContract`) also work with the `"NFTBridge"` module since the token bridge and nft bridge handle the same governance message payloads